google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/instance/AppEngineServerModel.java

```diff
@@ -174,11 +173,6 @@ public void checkConfiguration() throws RuntimeConfigurationException {
           .sdkPath(sdkService.getSdkHomePath())
           .build();
       sdk.validateCloudSdk();
```

Comment:
We can probably go back to not instantiating the sdk variable and calling validateCloudSdk() after build().